Packs/SentryWire/Integrations/SentryWire/SentryWire.yml

```
@@ -0,0 +1,258 @@
+      default: false
+      isArray: false
+    deprecated: false
+    description: Download search pcap packaged in a zip file.
+    execution: false
+    name: sentrywire-get-pcap
+    outputs:
+    - contextPath: File.Size
+      description: The size of the file.
+      type: Number
+    - contextPath: File.SHA1
+      description: The SHA1 hash of the file.
+      type: String
+    - contextPath: File.SHA256
+      description: The SHA256 hash of the file.
+      type: String
+    - contextPath: File.Name
+      description: The name of the file.
+      type: String
+    - contextPath: File.SSDeep
+      description: The SSDeep hash of the file.
+      type: String
+    - contextPath: File.EntryID
+      description: The entry ID of the file.
+      type: String
+    - contextPath: File.Info
+      description: File information.
+      type: String
+    - contextPath: File.Type
+      description: The file type.
+      type: String
+    - contextPath: File.MD5
+      description: The MD5 hash of the file.
+      type: String
+    - contextPath: File.Extension
+      description: The file extension.
+      type: String
+  - arguments:
+    - name: search_id
+      required: true
+      secret: false
+      description: The ID of the search.
+      default: false
+      isArray: false
+    - name: node_name
+      required: true
+      secret: false
+      description: Target node to check search status.
+      default: false
+      isArray: false
+    deprecated: false
+    description: Get the current status of a search.
+    execution: false
+    name: sentrywire-get-search-status
+    outputs:
+    - contextPath: SentryWire.Investigator.Begintime
+      description: The earliest date time a search should target (ISO 8601 format) UTC time - YYYY-MM-DD hh:mm:ss.
+      type: String
+    - contextPath: SentryWire.Investigator.Endtime
+      description: The latest date time a search should target (ISO 8601 format) UTC time - YYYY-MM-DD hh:mm:ss.
+      type: String
+    - contextPath: SentryWire.Investigator.MaxPacketCount
+      description: Maximum number of packets.
+      type: String
+    - contextPath: SentryWire.Investigator.NodeName
+      description: Node where status of search is being checked.
+      type: String
+    - contextPath: SentryWire.Investigator.SearchFilter
+      description: The filter being used by the search.
+      type: String
+    - contextPath: SentryWire.Investigator.SearchID
+      description: The ID of the search.
+      type: String
+    - contextPath: SentryWire.Investigator.SearchResult
+      description: Results of the search being run.
+      type: String
+    - contextPath: SentryWire.Investigator.SearchStatus
+      description: Status of the search pending/completed/cancelled.
+      type: String
+  - arguments:
+    - default: false
+      isArray: false
+    - description: ID of search to delete.
+      name: search_id
+      required: true
+      secret: false
```

You have an issue in the yml here why did you open a new section there is only one parameter search_id

```suggestion
      default: false
      isArray: false
      description: ID of search to delete.
    - name: search_id
      required: true
      secret: false
```

---

This was probably an oversight when this yml was written.